docs/docs/configuration/auth.md

```diff
@@ -87,6 +88,21 @@ Note: The user is checked against the group members list on initial authenticati
 
 Note: When using the Azure Auth provider with nginx and the cookie session store you may find the cookie is too large and doesn't get passed through correctly. Increasing the proxy_buffer_size in nginx or implementing the [redis session storage](sessions.md#redis-storage) should resolve this.
 
+### Azure Auth Provider
+
+1. Open the AD FS administration console on your Windows Server and add a new Application Group
```

Comment:
Can we consolidate on how we refer to ADFS in the docs, we have a few different case structure in this writeup for how we refer to it.

Is ADFS the appropriate name throughout?